_quarto.yml

```diff
@@ -1,6 +1,10 @@
 project:
   type: website
   output-dir: docs
+  # For some reasons the logo light isn't properly copied
+  # so we have to do it post-render.
+  post-render: 
+    - "cp ./logo_light.svg docs/"
```

Comment:
If there's a better way to quarto to copy tell logo_light.svg. Using `resources` does not work: https://quarto.org/docs/websites/website-tools.html#site-resources. At least that works.

Reply:
I think this is smart!